test/unix/common.ml

```diff
@@ -188,3 +188,48 @@ let check_disjoint index htbl =
           Alcotest.failf "Found value %a when checking for the absence of %a"
             (Repr.pp Value.t) v' pp_binding (k, v))
     htbl
+
+let get_open_fd root =
```

Comment:
there is a bit of code duplication with `test_fd` in the force_merge tests, can you maybe combine the two functions?

Reply:
Done.  I replaced the `/tmp` directory used to look for the files with `root` since all the files are created there from what I can see